src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFolderProperty.java

```diff
@@ -150,6 +152,7 @@ public UsagePermission(String name, boolean granted, boolean inherited) {
             this.inherited = inherited;
         }
 
+        @SuppressFBWarnings("UPM_UNCALLED_PRIVATE_METHOD")
```

Comment:
These methods should be deleted instead. Unused private methods are dead code.

Reply:
Sure, I removed them.